src/LayoutManager/LayoutManager.ts

```diff
@@ -248,6 +250,8 @@ export class LayoutManager {
       left: object.left + offset.x,
       top: object.top + offset.y,
     });
+    // TODO: should we delete aCoords/oCoords here so they are not stale if subTargetCheck is false?
+    //  or should we apply offset instead of recalculating aCoords?
```

Comment:
an object without coords could create more damages than one with stale, different typings, always have to check if the coords are there.
We should document that if you have a group without subtargetCheck and you plan on using the group inner object coordinates, you have to update them when you use them.